omod-2.1/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_1/CohortMembershipResource2_1.java

```diff
@@ -93,6 +97,36 @@ public DelegatingResourceDescription getUpdatableProperties() throws ResourceDoe
 		return d;
 	}
 	
+	@Override
+	public Model getGETModel(Representation rep) {
+		ModelImpl model = (ModelImpl) super.getGETModel(rep);
+		if (rep instanceof DefaultRepresentation || rep instanceof FullRepresentation) {
+			model
+			        .property("uuid", new StringProperty())
+			        .property("display", new StringProperty())
+			        .property("startDate", new DateProperty())
+			        .property("endDate", new DateProperty())
+			        .property("patientUuid", new StringProperty());
+		}
+		//FIXME missing props
```

Comment:
@dkayiwa IMO. lines marked FIXME that'll update the swagger properties will be better fix in a new separate issue.